src/cmd/cli/command/commands.go

```diff
@@ -357,7 +357,7 @@ var RootCmd = &cobra.Command{
 				term.Debug("Server error:", err)
 				term.Warn("Please log in to continue.")
 
-				defer trackCmd(nil, "Login", P{"reason", err})
+				defer func() { trackCmd(nil, "Login", P{"reason", err}) }()
```

Comment:
this was actually a bug: it was supposed to track the error returned by `cli.InteractiveLogin` but would track the error as it is when this line hits, not when the `defer` triggers.

Reply:
Oof that's tricky. Should we just call it inline after the call to `cli.InteractiveLogin` then?